### ISSUE TITLE
Fatal error: require_once() ... wp-content/plugins/woocommerce-admin//includes/feature-config.php

### ISSUE THREAD

**Issue by allendav** [woocommerce/woocommerce-admin#2053]
**Describe the bug**
Fatal error on activating the plugin after a fresh cloning and `npm run build`.

**To Reproduce**

1. Clone the repo
2. `npm run build`
3. See this during the build process

```
@woocommerce/admin-library@0.10.0 build:feature-config /Applications/MAMP/htdocs/zzz/wp-content/plugins/woocommerce-admin
> > php bin/generate-feature-config.php
> Warning: fopen(dist/feature-config-core.php): failed to open stream: No such file or directory in /Applications/MAMP/htdocs/zzz/wp-content/plugins/woocommerce-admin/bin/generate-feature-config.php on line 38
> Warning: fwrite() expects parameter 1 to be resource, boolean given in /Applications/MAMP/htdocs/zzz/wp-content/plugins/woocommerce-admin/bin/generate-feature-config.php on line 41
> Warning: fclose() expects parameter 1 to be resource, boolean given in /Applications/MAMP/htdocs/zzz/wp-content/plugins/woocommerce-admin/bin/generate-feature-config.php on line 42
```

and then see this on attempting to activate the plugin:

```
Warning: require_once(/Applications/MAMP/htdocs/zzz/wp-content/plugins/woocommerce-admin//includes/feature-config.php): failed to open stream: No such file or directory in /Applications/MAMP/htdocs/zzz/wp-content/plugins/woocommerce-admin/woocommerce-admin.php on line 233

Fatal error: require_once(): Failed opening required '/Applications/MAMP/htdocs/zzz/wp-content/plugins/woocommerce-admin//includes/feature-config.php' (include_path='.:/Applications/MAMP/bin/php/php7.2.10/lib/php') in /Applications/MAMP/htdocs/zzz/wp-content/plugins/woocommerce-admin/woocommerce-admin.php on line 233
```

**Expected behavior**

Error free activation. :)

**Desktop (please complete the following information):**

* Mojave, Chrome, MAMP

**Comment by psealock** [comment]
This is affecting `npm run build:release` as well, which is preventing us from releasing. 

Interestingly, `npm start` before `npm run build` avoids the error above, but the js app never runs, despite being added to page.

I've tried to playing with the order of tasks to match `npm start` with no luck and I've ensured the `WC_ADMIN_PHASE` environment variable is correctly passed around which it appears to be.

**Comment by psealock** [comment]
Actually, this issue has been resolved by https://github.com/woocommerce/woocommerce-admin/pull/2046. If we update the main readme, we can close this one.

**Comment by ryelle** [comment]
> Actually, this issue has been resolved by #2046. If we update the main readme, we can close this one.

It hasn't – `npm run dev` doesn't have the same issue because it's running in the development env, so it creates `includes/feature-config.php `. If you run the following, you still see this issue:

```
npm run clean && rm includes/feature-config.php && npm run build
```

I don't feel confident enough in why we'd want `dist/feature-config.php` _and_ `includes/feature-config.php` to debug further though.

**Comment by justinshreve** [comment]
> I don't feel confident enough in why we'd want dist/feature-config.php and includes/feature-config.php to debug further though.

I wrote a reply here that kinda covers it:
https://github.com/woocommerce/woocommerce-admin/pull/2115#pullrequestreview-230816648

We can revisit that decision.
